docs/nodes/validator-node.md

```diff
@@ -279,8 +279,8 @@ You have successfully set up a bridge node that is syncing with the network.
 
 ### Setup QGB keys
 
-This step helps get you prepared for when the Quantum Gravity Bridge
-is ready to be deployed. You would still need to go through this step
+This step helps get you prepared for when the Quantum Gravity Bridge relayer
+is deployed by Celestia Labs. You would still need to go through this step
```

Comment:
After thinking about it, we can either leave it as is or replace with the following: 

```suggestion
This step helps get you prepared for when the Quantum Gravity Bridge
is enabled. You would still need to go through this step
```